lib/model.js

```diff
@@ -40,6 +40,9 @@ class Model extends Entity {
         ModelInstance.prototype.gstore = gstore;
         ModelInstance.gstore = gstore;
 
+        ModelInstance.prototype.parseId = getParseId(schema, gstore);
```

Comment:
As we use it like this `self.model.parseId` in  `entity.js` I don't think we need to add it to the prototype object.

Reply:
You're right, I'm removing those too.